azure-devops/run-build.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       inputs:
         cmakeListsOrSettingsJson: 'CMakeListsTxtAdvanced'
         cmakeListsTxtPath: '$(Build.SourcesDirectory)/CMakeLists.txt'
-        buildDirectory: $(Build.ArtifactStagingDirectory)/${{ parameters.targetPlatform }}
+        buildDirectory: D:/build/${{ parameters.targetPlatform }}

```

Comment:
I believe `D:/build` should be centralized in the `variables` at the top, like `vcpkgLocation`.